### PR TITLE
Add rubygem-foreman_kernel_care package

### PR DESCRIPTION
**Pull request by maccelf**
Add [foreman_kernel_care](https://github.com/maccelf/foreman_kernel_care) plugin

### CODE REVIEW

**Comment by ekohl** [comment]
ok to test

**Comment by Odilhao** [comment]
[test rpm]

**Comment by ares** [comment]
Is there anything blocking this PR? rpm seems to build
[test rpm] to get fresh results

File: comps/comps-foreman-plugins-el8.xml [hunk 1]
```diff
@@ -66,6 +66,7 @@
       <packagereq type="default">rubygem-foreman_hooks</packagereq>
       <packagereq type="default">rubygem-foreman_host_extra_validator</packagereq>
       <packagereq type="default">rubygem-foreman_host_reports</packagereq>
+      <packagereq type="default">rubygem-foreman_kernel_care</packagereq>
```
Comment (adamruzicka) [inline comps/comps-foreman-plugins-el8.xml:69]: This needs to go into `comps/comps-katello-el8.xml`

File: comps/comps-foreman-plugins-rhel7.xml [hunk 2]
```diff
@@ -65,6 +65,7 @@
       <packagereq type="default">tfm-rubygem-foreman_hooks</packagereq>
       <packagereq type="default">tfm-rubygem-foreman_host_extra_validator</packagereq>
       <packagereq type="default">tfm-rubygem-foreman_host_reports</packagereq>
+      <packagereq type="default">tfm-rubygem-foreman_kernel_care</packagereq>
```
Comment (adamruzicka) [inline comps/comps-foreman-plugins-rhel7.xml:68]: `comps/comps-katello-el8.xml`

File: package_manifest.yaml [hunk 3]
```diff
@@ -994,6 +994,7 @@ plugin_scl_packages:
     rubygem-foreman_hooks: {}
     rubygem-foreman_host_extra_validator: {}
     rubygem-foreman_host_reports: {}
+    rubygem-foreman_kernel_care: {}
```
Comment (adamruzicka) [inline package_manifest.yaml:997]: This needs to go in `katello_packages` section

File: rel-eng/tito.props [hunk 4]
```diff
@@ -685,6 +685,7 @@ whitelist = ansible-collection-theforeman-foreman
   rubygem-foreman_hooks
   rubygem-foreman_host_extra_validator
   rubygem-foreman_host_reports
+  rubygem-foreman_kernel_care
```
Comment (adamruzicka) [inline rel-eng/tito.props:688]: This needs to go into `katello-nightly-el{7,8}`

**Review by adamruzicka — CHANGES_REQUESTED** [review]
Because the plugin depends on Katello, it needs to be bundled in katello repos.

**Comment by adamruzicka** [comment]
Most likely the whole package directory needs to be in `packages/katello` instead of `packages/plugins`, sorry for not pointing that out previously

**Comment by adamruzicka** [comment]
As far as I can tell there is nothing more left to be done here. The failures seem to be caused by katello moving to a newer version of pulp_rpm_client in [1,2]

[1] - https://github.com/Katello/katello/pull/9976
[2] - https://github.com/theforeman/foreman-packaging/pull/7621

**Comment by maccelf** [comment]
@adamruzicka okay, thank you very much

**Comment by maccelf** [comment]
@adamruzicka now we are waiting for the release of Katello right?

**Comment by adamruzicka** [comment]
That's my understanding, yes

**Comment by maccelf** [comment]
@adamruzicka Now this PR can be merged ?

**Comment by adamruzicka** [comment]
I don't see why not, but I don't have necessary permissions here. @zjhuntin maybe?

**Comment by evgeni** [comment]
Doing the @theforeman/packaging job again…

```console
evgeni@tanso ~ % koji add-pkg --owner kojiadm katello-nightly-el8 rubygem-foreman_kernel_care         
Adding 1 packages to tag katello-nightly-el8
evgeni@tanso ~ % koji add-pkg --owner kojiadm katello-nightly-rhel7 tfm-rubygem-foreman_kernel_care 
Adding 1 packages to tag katello-nightly-rhel7
```

**Comment by maccelf** [comment]
Awesome, thanks very much

**Pull request merged**